app/src/main/java/com/quarkworks/apartmentgroceries/user/PhotoActivity.java

```diff
@@ -56,15 +72,20 @@ public void onClick(View v) {
         });
 
         SharedPreferences sharedPreferences = getSharedPreferences(getString(R.string.login_or_sign_up_session), 0);
-        String photoUrl = sharedPreferences.getString(SyncUser.JsonKeys.URL, null);
-
-        if (!TextUtils.isEmpty(photoUrl)) {
-            Glide.with(this)
-                    .load(photoUrl)
-                    .centerCrop()
-                    .crossFade()
-                    .into(photoImageView);
-        }
+        String usrId = sharedPreferences.getString(SyncUser.JsonKeys.USER_ID, null);
+
+        SyncUser.getById(usrId).continueWith(new Continuation<RUser, Void>() {
+            @Override
```

Comment:
Same issue as an earlier comment. You aren't setting the image until the network request ends.